json_feature.cpp

```diff
@@ -1,27 +1,38 @@
 
 #include "json_feature.hpp"
 
-void JSONFeature::add_properties(const osmium::OSMObject& object, const char* id_name) {
+void JSONFeature::add_properties(const osmium::OSMObject& object) {
     m_writer.String("properties");
 
     m_writer.StartObject();
 
-    m_writer.String(id_name);
-    m_writer.Int64(object.id());
+    m_writer.String((m_attr_prefix + "id").c_str());
```

Comment:
Per chat, we would need to do some more refactoring to make consolidating the concatenations work.

Also, I tried the `#define RAPIDJSON_HAS_STDSTRING=1` but didn't have any luck.